src/sentry/static/sentry/app/views/acceptOrganizationInvite.tsx

```diff
@@ -0,0 +1,220 @@
+    e.preventDefault();
+    await logout(this.api);
+    window.location.replace(this.makeNextUrl('/auth/login/'));
+  };
+
+  handleAcceptInvite = async () => {
+    const {memberId, token} = this.props.params;
+
+    this.setState({accepting: true});
+    try {
+      await this.api.requestPromise(`/accept-invite/${memberId}/${token}/`, {
+        method: 'POST',
+      });
+      browserHistory.replace(`/${this.state.inviteDetails.orgSlug}/`);
+    } catch {
+      this.setState({acceptError: true});
+    }
+    this.setState({accepting: false});
+  };
+
+  get existingMemberAlert() {
+    const user = ConfigStore.get('user');
+
+    return (
+      <Alert type="warning" data-test-id="existing-member">
+        {tct(
+          'Your account ([email]) is already a member of this organization. [switchLink:switch accounts]?',
+          {
+            email: user.email,
+            switchLink: <Link href="#" onClick={this.handleLogout} />,
+          }
+        )}
+      </Alert>
+    );
+  }
+
+  get authenticationActions() {
+    const {inviteDetails} = this.state;
+
+    return (
+      <React.Fragment>
+        <p>
+          {t(
+            `To continue, you must either login to an existing Sentry account,
+             or create a new account.`
+          )}
+        </p>
+        {inviteDetails.needsSso && (
+          <p data-test-id="suggests-sso">
+            {tct(
+              `Note that [orgSlug] has enabled Single-Sign-On (SSO) using
+               [authProvider]. You may create an account by authenticating with
+               the organizations SSO provider.`,
+              {
+                orgSlug: inviteDetails.orgSlug,
+                authProvider: inviteDetails.ssoProvider,
+              }
+            )}
+          </p>
+        )}
+
+        <Actions>
+          {inviteDetails.needsSso ? (
+            <Button
+              label="sso-login"
+              priority="primary"
+              href={this.makeNextUrl(`/auth/login/${inviteDetails.orgSlug}/`)}
+            >
+              {t('Login with %s', inviteDetails.ssoProvider)}
+            </Button>
+          ) : (
+            <Button
+              label="create-account"
+              priority="primary"
+              href={this.makeNextUrl('/auth/register/')}
+            >
+              {t('Create a new account')}
+            </Button>
+          )}
+          <Link href={this.makeNextUrl('/auth/login/')}>
+            {t('Login using an existing account')}
+          </Link>
+        </Actions>
+      </React.Fragment>
+    );
+  }
+
+  get warning2fa() {
+    const {inviteDetails} = this.state;
+
+    return (
+      <React.Fragment>
+        <p data-test-id="2fa-warning">
+          {tct(
+            'To continue, [orgSlug] requires all members to configure two-factor authentication.',
+            {orgSlug: inviteDetails.orgSlug}
+          )}
+        </p>
+        <Actions>
+          <Button priority="primary" to="/account/settings/2fa/">
```

Comment:
Can we update this to `/settings/account/security/`? In the require 2fa login flow, I'm getting an error on `/account/settings/2fa/` rather than being redirected.

Reply:
Ah, yeah